src/platform_impl/macos/window.rs

```diff
@@ -1441,7 +1441,7 @@ impl WindowExtMacOS for UnownedWindow {
 
   #[inline]
   fn ns_view(&self) -> *mut c_void {
-    *self.ns_view as *mut _
```

Comment:
I think we should remove the .ns_view field, it can lead to errors and it's referenced in other places too

Reply:
We can't really do this because the NSView subclass from wry doesn't have the taoState ivar. We get back to that issue about some missing tao events when using wry.